packages/itwinui-react/src/core/Tabs/Tabs.tsx

```diff
@@ -117,6 +138,14 @@ export type VerticalTabsProps = Omit<TabsProps, 'orientation' | 'type'> & {
  * <Tabs labels={tabsWithIcons} type='pill' />
  */
 export const Tabs = (props: TabsProps) => {
+  // Separate actions from props to avoid adding it to the DOM (using {...rest})
+  let actions: Array<React.ReactNode> | undefined = undefined;
```

Comment:
it should already default to undefined

```suggestion
  let actions: Array<React.ReactNode>;
```

Reply:
seems you applied the suggestion before my edit. it defaults to undefined but you still need to make TS happy.
```
let actions: Array<React.ReactNode> | undefined;
```
or just use ReactNode
```
let actions: React.ReactNode;
```